test/cpp/qps/qps_worker.h

```diff
@@ -39,11 +39,13 @@ extern std::vector<grpc::testing::Server*>* g_inproc_servers;
 class QpsWorker {
  public:
   explicit QpsWorker(int driver_port, int server_port,
-                     const std::string& credential_type);
+                     const std::string& credential_type,
+                     bool die_on_connection_failure);
```

Comment:
nit: maybe set this as a default parameter to `true` so that we don't need to update other call sites

Reply:
Done.